docs/DataSHIELD-datamanagement/2_Creating_data_subsets_Armadillo.r

```diff
@@ -80,7 +79,7 @@ armadillo.upload_table(study, "2_1-core-1_0", subset_core_yearlyrep, "yearlyrep"
 # See if tables are uploaded
 armadillo.list_tables(study)
 
-# Now you can also take a look at the files in the user interface of the MinIO file server. In this case: https://armadillo-minio.test.molgenis.org
+# Now you can also take a look at the files in the user interface of the Armadillo server. In this case: https://armadillo-minio.test.molgenis.org
```

Comment:
There is still a reference to minio

Reply:
To have a look at the files you can use the Armadillo UI projects: https://molgenis.github.io/molgenis-service-armadillo/#/ui?id=project-explorer-
So in this case the url would be https://armadillo.test.molgenis.org/#/projects

Reply:
Done